buildenv/docker/mkdocker.sh

```diff
@@ -219,12 +220,22 @@ if [ $cuda != no ] ; then
   echo "FROM nvidia/cuda:${cuda}-devel-ubuntu16.04 AS cuda-dev"
   echo ""
 fi
+if [ $dist = centos ] ; then
+  echo "FROM $dist:$dist$version"
+else
   echo "FROM $dist:$version"
+fi
 }
 
 install_centos_packages() {
+if [ $version = 6.10 ] ; then
+  echo "RUN sed -i -e 's|mirrorlist|#mirrorlist|' \\"
+  echo "           -e 's|#baseurl=http://mirror.centos.org/centos/\$releasever|baseurl=https://vault.centos.org/6.10|' \\"
+  echo "           /etc/yum.repos.d/CentOS-Base.repo"
+  echo ""
+fi
```

Comment:
Why is this needed? Don't the default repo configurations work?

Reply:
When I initially did the tag change and repo change was what I needed to get this functional.  I'll try a run with this and the repo reverted.

Reply:
CentOS 6 is no longer hosted at mirror.centos.org as per http://mirror.centos.org/centos/6/readme so I modify the system default to redirected to the vault/archive location.

Reply:
Why are we not heeding the warning in that readme file?
> The whole CentOS 6 is *dead* and *shouldn't* be used anywhere at *all*

Reply:
We are still using as this is the operating system that is used at Eclipse Adoptium for JDK8 x86(-64) support.

Reply:
Not just jdk8, but jdk11+ as well.